sunpy/net/dataretriever/sources/rhessi.py

```diff
@@ -62,8 +62,8 @@ def get_observing_summary_filename(self, time_range):
         --------
         >>> from sunpy.net.dataretriever.sources.rhessi import RHESSIClient
         >>> RHESSIClient().get_observing_summary_filename(('2011/04/04', '2011/04/05'))   # doctest: +REMOTE_DATA
-        ['https://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/hsi_obssumm_20110404_042.fits',
-        'https://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/hsi_obssumm_20110405_031.fits']
+        ['https://.../hessidata/metadata/catalog/hsi_obssumm_20110404_042.fits',
+        'https://.../hessidata/metadata/catalog/hsi_obssumm_20110405_031.fits']
```

Comment:
I thought of that but then I was like, it makes the example opaque.

Reply:
Yeah, it's not great but it's better than skipping it.